org.ekstep.lessonbrowser-1.4/editor/lessonBrowserApp.js

```diff
@@ -422,7 +422,8 @@ angular.module('org.ekstep.lessonbrowserapp', ['angular-inview', 'luegg.directiv
                         var contents = [];
                         $scope.mainTemplate = 'facetsItemView';
                         ctrl.applyAllJquery();
-                        if (ctrl.facetsResponse.result.response.sections) {
+                        var contentCount = _.findIndex(ctrl.facetsResponse.result.response.sections, function(section) { return section.count > 0 });
+                        if (ctrl.facetsResponse.result.response.sections && contentCount > 0) {
```

Comment:
Define an object for ctrl.facetsResponse.result.response.sections & use that object.